fdbserver/Status.actor.cpp

```diff
@@ -1298,32 +1298,26 @@ ACTOR static Future<JsonBuilderObject> dataStatusFetcher(WorkerDetails ddWorker,
 				stateSectionObj["healthy"] = false;
 				stateSectionObj["name"] = "healing";
 				stateSectionObj["description"] = "Restoring replication factor";
-			}
-			else if (highestPriority >= PRIORITY_TEAM_REDUNDANT) {
-				stateSectionObj["healthy"] = true;
-				stateSectionObj["name"] = "optimizing_team_collections";
-				stateSectionObj["description"] = "Optimizing team collections";
-			}
-			else if (highestPriority >= PRIORITY_MERGE_SHARD) {
+			} else if (highestPriority >= PRIORITY_MERGE_SHARD) {
 				stateSectionObj["healthy"] = true;
 				stateSectionObj["name"] = "healthy_repartitioning";
 				stateSectionObj["description"] = "Repartitioning.";
-			}
-			else if (highestPriority >= PRIORITY_TEAM_CONTAINS_UNDESIRED_SERVER) {
+			} else if (highestPriority >= PRIORITY_TEAM_REDUNDANT) {
+				stateSectionObj["healthy"] = true;
+				stateSectionObj["name"] = "optimizing_team_collections";
+				stateSectionObj["description"] = "Optimizing team collections";
+			} else if (highestPriority >= PRIORITY_TEAM_CONTAINS_UNDESIRED_SERVER) {
 				stateSectionObj["healthy"] = true;
 				stateSectionObj["name"] = "healthy_removing_server";
 				stateSectionObj["description"] = "Removing storage server";
```

Comment:
This message looks confusing/scary. Maybe something like "optimizing team servers"?

Reply:
This is for `PRIORITY_TEAM_CONTAINS_UNDESIRED_SERVER` which is not touched by this PR.

I'm inclined to keep what it is because undesired server (e.g., the server on the same address with another server) is supposed to be removed.